e2e/iam/project_teams_test.go

```diff
@@ -0,0 +1,162 @@
+)
+
+func TestProjectTeams(t *testing.T) {
+	cliPath, err := e2e.Bin()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	n, err := e2e.RandInt(1000)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	projectName := fmt.Sprintf("e2e-proj-%v", n)
+	projectID, err := createProject(projectName)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	teamName := fmt.Sprintf("e2e-teams-%v", n)
+	teamID, err := createTeam(teamName)
+
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	defer func() {
+		if e := deleteProject(projectID); e != nil {
+			t.Errorf("error deleting project: %v", e)
+		}
+		if e := deleteTeam(teamID); e != nil {
+			t.Errorf("error deleting team: %v", e)
+		}
+	}()
+
+	t.Run("Add", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			iamEntity,
+			projectEntity,
+			teamsEntity,
+			"add",
+			teamID,
+			"--role",
+			"GROUP_READ_ONLY",
+			"--projectId",
+			projectID,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		assert.NoError(t, err, string(resp))
+
+		var teams mongodbatlas.TeamsAssigned
+		if err = json.Unmarshal(resp, &teams); err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+
+		found := false
+		for _, team := range teams.Results {
+			if team.TeamID == teamID {
+				found = true
+				break
+			}
+		}
+
+		assert.True(t, found)
+	})
+
+	t.Run("Update", func(t *testing.T) {
+		roleName1 := "GROUP_READ_ONLY"
+		roleName2 := "GROUP_DATA_ACCESS_READ_ONLY"
+		cmd := exec.Command(cliPath,
+			iamEntity,
+			projectEntity,
+			teamsEntity,
+			"update",
+			teamID,
+			"--role",
+			roleName1,
+			"--role",
+			roleName2,
+			"--projectId",
+			projectID,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		assert.NoError(t, err, string(resp))
+
+		var roles []mongodbatlas.TeamRoles
+		if err = json.Unmarshal(resp, &roles); err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+
+		assert.Len(t, roles, 1)
+
+		role := roles[0]
+
+		assert.Equal(t, role.TeamID, teamID)
+		assert.Len(t, role.RoleNames, 2)
+		for _, roleName := range role.RoleNames {
+			assert.Contains(t, []string{roleName1, roleName2}, roleName)
+		}
+	})
+
+	t.Run("List", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			iamEntity,
+			projectEntity,
+			teamsEntity,
+			"ls",
+			"--projectId",
+			projectID,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		assert.NoError(t, err, string(resp))
+
+		var teams mongodbatlas.TeamsAssigned
+		if err = json.Unmarshal(resp, &teams); err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+
+		assert.NotEmpty(t, teams.Results)
+	})
+
+	t.Run("Delete", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			iamEntity,
+			projectEntity,
+			teamsEntity,
+			"delete",
+			teamID,
+			"--force",
+			"--projectId",
+			projectID)
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		assert.NoError(t, err, string(resp))
```

Comment:
Please add an assertion for the message, to confirm the output, we do this for all our `delete`s and has help us to discover bugs